ocaml/xapi/xapi_psr.ml

```diff
@@ -26,6 +29,23 @@ type failure =
 
 type 'a r = (unit, failure * 'a) result
 
+let user_facing_error_message ~__context error =
+  let failure, self = error in
+  let host_name = Db.Host.get_name_label ~__context ~self in
+  let try_again_msg = "Please check the management network and try again." in
+  match failure with
+  | Failed_during_accept_new_pool_secret ->
+      Printf.sprintf "%s refused to start accepting new pool secret. %s"
+        host_name try_again_msg
+  | Failed_during_send_new_pool_secret ->
+      Printf.sprintf "%s refused to start sending new pool secret. %s" host_name
```

Comment:
The messages seem too convoluted. I think they can be made simpler while not losing meaning, like `%s refused the new pool secret. %s` and `%s refused to send a new pool secret. %s`.

Reply:
'refused the new pool secret' implies that the host refused some request because of a pool secret, but that isn't what is going on here. What I mean is that the operation to tell the host to start accepting the new pool secret failed (and similarly for the other message).

I'll change it to `operation to tell $host to begin accepting new pool secret failed`